### DISCUSSION TITLE
Is blocking maximum of resources?

### DISCUSSION THREAD

**Discussion by zipus** [frePPLe/frepple#610] [Q&A] [answered]
I read some time ago that when handling resources (employees), a good approach is to have a general resource assigned to the step when multiple people can do various tasks based on the skills.
So, I created a resource called Generic Employee, and loaded the real employees with Generic Employee as a category for them, so the structure is something like:
```
Resource 1. Generic Employee, maximum: 1
Resource 2. Johan, maximum 1, category = resource 1
Resource 3. Zipus, maximum 1, category = resource 1
And then created the skills for resources 2 and 3.
```

Let's say that Zipus, due to the nature of the skills he executes can handle 2 operations at a time. So I update to:
`Resource 3. Zipus, maximum 2, category = resource 1`

Will the resource 1 (as has max 1) be blocking that the resource 3 can handle more than 1?

**Comment by jdetaeye** [comment, marked as answer]
I assume you set the "owner" field rather than the "category" field to define that Zipus and Johan are part of the "generic employee" group.

>Will the resource 1 (as has max 1) be blocking that the resource 3 can handle more than 1?

No, it won't block things.   
The algorithm works with the individual members in the group, and uses their maximum size, efficiency and available calendar.
The "generic employee" is just a dummy/aggregate resource.   We don't use the size, efficiency or calendar populated at the aggregate level.
